Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -173,7 +174,12 @@
 _phase2_globalValidation = _run3_globalValidation.copy()
 _phase2_globalValidation += me0SimValid
 
+_phase_1_globalValidation = globalValidation.copy()
+_phase_1_globalValidation += siPixelPhase1OfflineDQM_sourceV
```

Comment:
Hi @davidcarbonis @dmitrijus - sorry for my slow followup - so this PR seems not to replace the phase0 validation, but just to add to it a new one. So two questions, 1) what is the plan to replace? and 2) how many histogram bins are added (in light also of the increase seen in Tier0)